modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/insertbutton/plugin.js

```diff
@@ -192,7 +207,10 @@
 		_onButtonClick(event) {
 			event.cancel();
 
-			this._createToolbar();
+			// `event.editor` is undefined :/
```

Comment:
I tried for a long time investigate the undefined issue here and I decided to maintain this.editor always updated doing `this.editor = editor` in some event handlers that has a filled value for event.editor :)